client/core/core.go

```diff
@@ -2333,18 +2371,33 @@ func (c *Core) MaxSell(host string, base, quote uint32) (*OrderEstimate, error)
 	if err != nil {
 		return nil, fmt.Errorf("%s wallet MaxOrder error: %v", unbip(base), err)
 	}
-	sellRedemptionPer, err := quoteWallet.RedemptionFees()
+
+	dc, _, err := c.dex(host)
 	if err != nil {
-		return nil, fmt.Errorf("%s RedemptionFees error: %v", unbip(quote), err)
+		return nil, err
 	}
 
-	return &OrderEstimate{
-		Lots:           maxSell.Lots,
-		Value:          maxSell.Value,
-		MaxFees:        maxSell.MaxFees,
-		EstimatedFees:  maxSell.EstimatedFees,
-		Locked:         maxSell.Locked,
-		RedemptionFees: maxSell.Lots * sellRedemptionPer,
+	// Estimate a quote-converted lot size.
+	lotSize := quoteAsset.LotSize
+	book, found := dc.books[marketName(base, quote)]
+	if found {
+		midGap, err := book.MidGap()
+		if err == nil {
+			lotSize = calc.BaseToQuote(midGap, baseAsset.LotSize)
+		}
+	}
```

Comment:
Would you comment on the meaning of `!found`?  Looks like if there's no book for the base-quote pair, then the quote asset's lot size is used instead of a "quote-converted" lots size based on current exchange rates.  I'm wondering about the utility of this case, or if it should be an error considering the `MaxSell` caller is giving a base-quote pair they intend to trade on.

Reply:
You've convinced me. Error is better.